win/rl/winrl.cc

```diff
@@ -402,6 +409,19 @@ NetHackRL::fill_obs(nle_obs *obs)
             obs->inv_oclasses[i] = MAXOCLASSES;
         }
     }
+    if (obs->glyph_strs) {
+        int i = 0;
+        for (const glyph_str_t  &glyph_str : glyph_strs_) {
+            bool is_word_done = false;
+            for (int j = 0; j < NLE_GLYPH_STR_LENGTH; ++j) {
+                if (glyph_str[j] == '\0') {
+                    is_word_done = true;
+                }
+                obs->glyph_strs[i++] = is_word_done ? 0 : glyph_str[j];
```

Comment:
I'm not 100% certain what this does. It appears to assume that `glyph_str` is not only null-terminated but that any null will be followed by nulls only?